Firestore/core/src/firebase/firestore/remote/exponential_backoff.h

```diff
@@ -93,20 +99,19 @@ class ExponentialBackoff {
   }
 
  private:
-  using Milliseconds = util::AsyncQueue::Milliseconds;
```

Comment:
This private type alias was fine. The problem was with putting it at the top level outside the class.